tests/TopDownProteomics.Tests/ProForma/ProFormaParserTests.cs

```diff
@@ -89,18 +89,19 @@ public void MultipleDescriptorTag()
         }
 
         [Test]
-        public void ValueOnlyDescriptor()
+        [TestCase("PRO[Methyl]TEOFORM", "PROTEOFORM", "Methyl")]
+        [TestCase("PRO[Fe[III]]TEOFORM", "PROTEOFORM", "Fe[III]")]
```

Comment:
Shouldn't this be `Cation:Fe[III]`? It begs the question for interim names with colons: should `[Cation:Fe[III]]` be allowed or should we force `[Unimod:Cation:Fe[III]]`?

Reply:
Good point. I think the intention in ProForma v1 is that we would allow only a small set of keys, and so "Cation:" wouldn't be considered a key.